keps/sig-windows/2258-node-log-query/README.md

```diff
@@ -287,15 +286,42 @@ a 404 will be returned.
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+A rollout or rollback will not fail either when enabling or disabling the feature gate.
+The feature has no impact on running workloads.
+
+###### What specific metrics should inform a rollback?
+There are no specific metrics that inform a rollback other than the feature gate option
```

Comment:
A few possibilities: kubelet crashes? kubelet memory consumption goes really high (memory leak in the API, for example)? kubelet CPU utilization spikes.